Description: Use static tools on IA64
 On IA64, the shared binaries do not work. This is a partial revert of
 https://salsa.debian.org/kernel-team/klibc/commit/07809ffd86e1e4ca5a4e98e4708a1c1fae10babf
Author: Maximilian Attems <maks@debian.org>, Thorsten Glaser <tg@debian.org>, Frank Scheiner <frank.scheiner@web.de>
Last-Update: 2018-06-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/usr/dash/Kbuild
+++ b/usr/dash/Kbuild
@@ -93,4 +93,8 @@
 	$(Q):
 
 # Targets to install
-install-y := sh.shared
+ifeq ($(ARCH),ia64)
+      install-y := sh
+else
+      install-y := sh.shared
+endif
--- a/usr/kinit/fstype/Kbuild
+++ b/usr/kinit/fstype/Kbuild
@@ -22,4 +22,8 @@
 clean-dirs := static shared
 
 # install binary
-install-y := $(shared-y)
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y)
+else
+      install-y := $(shared-y)
+endif
--- a/usr/kinit/ipconfig/Kbuild
+++ b/usr/kinit/ipconfig/Kbuild
@@ -28,4 +28,8 @@
 clean-dirs := static shared
 
 # install binary
-install-y := $(shared-y)
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y)
+else
+      install-y := $(shared-y)
+endif
--- a/usr/kinit/nfsmount/Kbuild
+++ b/usr/kinit/nfsmount/Kbuild
@@ -24,4 +24,8 @@
 clean-dirs := static shared
 
 # Install binary
-install-y := $(shared-y)
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y)
+else
+      install-y := $(shared-y)
+endif
--- a/usr/kinit/resume/Kbuild
+++ b/usr/kinit/resume/Kbuild
@@ -27,4 +27,8 @@
 clean-dirs := static shared
 
 # install binary
-install-y := $(shared-y)
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y)
+else
+      install-y := $(shared-y)
+endif
--- a/usr/kinit/run-init/Kbuild
+++ b/usr/kinit/run-init/Kbuild
@@ -31,4 +31,8 @@
 clean-dirs := static shared
 
 # install binary
-install-y := $(shared-y)
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y)
+else
+      install-y := $(shared-y)
+endif
--- a/usr/utils/Kbuild
+++ b/usr/utils/Kbuild
@@ -73,4 +73,8 @@
 clean-dirs := static shared
 
 # install only install the shared binaries
-install-y := $(shared-y) shared/reboot shared/poweroff
+ifeq ($(ARCH),ia64)
+      install-y := $(static-y) static/reboot static/poweroff
+else
+      install-y := $(shared-y) shared/reboot shared/poweroff
+endif
